dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-child-node.ts

```diff
@@ -59,32 +63,79 @@ export function useChildNode({
     model.definition = res
   }
 
+  const getAccessibleProjectList = async () => {
+    queryProjectListPaging({
+      pageNo: 1,
+      pageSize: 999
+    })
```

Comment:
I think it's better to add api to query all projects for user.

Reply:
ok， will update later